client/ayon_core/tools/loader/models/actions.py

```diff
@@ -737,12 +781,26 @@ def _load_representations_by_loader(self, loader, repre_contexts, options):
             if version < 0:
                 version = "Hero"
             try:
+                for hook_plugin in hooks.get("pre", []):
+                    hook_plugin.process(
+                        loader,
+                        repre_context,
+                        options=options
```

Comment:
The hook logic now seems to run in the UI - whereas it should most likely _actually_ trigger in the load API itself, right?

So that if you were to programmatically load or (or update?) that it'd also trigger the revelant hooks (or callbacks or what you may call it.)

Reply:
Moved it out to utils.